### ISSUE TITLE
Leave staging project in testing on pending checks

### ISSUE THREAD

**Issue by coolo** [openSUSE/open-build-service#6386]
If there is a build failure, the staging project is reported as 'failed' even if there are pending checks. As some of these checks may actually change the sources, this might be a temporary failure.

As such the project shouldnt' be treated as final - which is indicated by failure. The idea back when we implemented staging dashboard was to avoid expensive openqa queries for projects that require different work anyway. But this is obsolete by the status reports.

**Comment by lnussel** [comment]
yes, nevertheless still makes sense to visualize potential trouble early without showing a "final" state

**Comment by coolo** [comment]
I'm just talking about the overall state
